rpcserver.go

```diff
@@ -936,12 +941,126 @@ func (r *rpcServer) VerifyProof(ctx context.Context,
 	)
 	valid := err == nil
 
-	// TODO(roasbeef): also show additional final resting anchor
-	// information, etc?
+	decodedProof, err := marshalProofFile(proofFile, -1)
+	if err != nil {
+		return nil, fmt.Errorf("unable to marshal proof: %w", err)
+	}
+
+	return &tarorpc.VerifyProofResponse{
```

Comment:
right exactly, full proof is verified, but we only return the latest proof

not sure if we want to include an optional for previous_witnesses or a similar bool name. I was worried that it could become a massive response tho if the asset has deep history